packages/tests/src/reya_network/DistributeFunds.s.sol

```diff
@@ -0,0 +1,60 @@
+pragma solidity >=0.8.19 <0.9.0;
+
+import "forge-std/Test.sol";
+import { Script } from "forge-std/Script.sol";
+import { IPassivePoolProxy } from "../interfaces/IPassivePoolProxy.sol";
+import { IERC20TokenModule } from "../interfaces/IERC20TokenModule.sol";
+import { IRUSDProxy } from "../interfaces/IRUSDProxy.sol";
+
+contract DistributeFunds is Script, Test {
+  address multisigEOA = 0x01A8e78B7ba1313A482630837c3978c6259aC1eA;
+
+  address payable pool = payable(0xB4B77d6180cc14472A9a7BDFF01cc2459368D413);
+  address usdc = 0x3B860c0b53f2e8bd5264AA7c3451d41263C933F2;
```

Comment:
i have another address for USDC. in my env vars, 0x1b700A40A0707Dd5f0BC31cbc57D730558E5714d
can you double check pls?

Reply:
that's on testnet